### DISCUSSION TITLE
How to convert the .obj files to .bgeo files?

### DISCUSSION THREAD

**Discussion by ZF-Guo** [InteractiveComputerGraphics/SPlisHSPlasH#181] [Q&A]
Hi,
I can get the model in .obj file format through 3D drawing software（UG, PRO/E ）, but this model can only be analyzed as a rigid body. So, I would like to consult how to get a .bgeo file with particle information?
I found that after importing graphics drawn in mm units, the units became inches.

Best wishes,

**Comment by janbender** [comment]
HI, 

you can also use an OBJ file as FluidModel:

"FluidModels": [
		{
			"particleFile": "../models/bunny.obj",
			"scale": [1, 1, 1]
		}
	]

Then the geometry is sampled with particles. However, the geometry must be closed and manifold. 
Alternatively, you can use the tool VolumeSampling which is a part of SPlisHSPlasH. 

Hope that helps!

**Reply by ZF-Guo** [reply]
Thank you very much for your reply.

**Comment by janbender** [comment]
Concerning units: The unit for distances are meters.
